### ISSUE TITLE
long category titles in LSB incorrectly layout

### ISSUE THREAD

**Issue by bkmartinjr** [chanzuckerberg/cellxgene#1161]
When a category name is long (>40 or 50 characters) it will cause the LSB to layout incorrectly.   The droplet icon is not visible, and the "expand category" chevron may wrap.

These categories are fairly common in datasets from the HCA.

Ideally we would truncate the name using the same method used elsewhere (ie, clip middle of string, replacing it with an ellipsis, and use a tooltip to show the entire name).

Screen shot:

![image.png](https://images.zenhubusercontent.com/5af38d3b4b5806bc2bd28ea9/ea99cd4d-8c71-439f-b629-0559db05efa5)

**Comment by bkmartinjr** [comment]
fixed by #1206

**Comment by liaprins-czi** [comment]
This is happening for me again. I noticed it when testing a PR for Matt, but then I checked master and it was exactly the same there:

![Screen Shot 2020-04-27 at 9 44 05 AM](https://user-images.githubusercontent.com/53277128/80398931-42197980-886d-11ea-8794-4f182c9c5bf6.png)

Possibly it was just this combination of character widths? The max character count may need to be reduced by 1 or 2 for combinations like this?
